Makefile

```diff
@@ -91,11 +91,11 @@ build-plain:
 
 .PHONY: build-image
 build-image:
-	GOFLAGS="$(GO_FLAGS)" ko publish --preserve-import-paths ./cmd/manager
+	KO_DOCKER_REPO="$(IMAGE_HOST)/$(IMAGE)" GOFLAGS="$(GO_FLAGS)" ko publish --bare ./cmd/manager
 
 .PHONY: build-image-with-pprof
 build-image-with-pprof:
-	GOFLAGS="$(GO_FLAGS) -tags=pprof_enabled" ko publish --preserve-import-paths --tags=pprof ./cmd/manager
+	KO_DOCKER_REPO="$(IMAGE_HOST)/$(IMAGE)" GOFLAGS="$(GO_FLAGS) -tags=pprof_enabled" ko publish --bare --tags=pprof ./cmd/manager
```

Comment:
ko has this admonition for the `--bare` flag [1]:

```
Whether to just use KO_DOCKER_REPO without additional context (will not work properly with --tags)
```

How can we work around this?

[1] https://github.com/google/ko/blob/82cabb40bae577ce3bc016e5939fd85889538e8b/pkg/commands/options/publish.go#L67-L68

Reply:
Looking through the code, I'm pretty sure `--bare` _does_ work with tags.

`--bare` configures a `bareDockerNamer` [here](https://github.com/google/ko/blob/v0.8.0/pkg/commands/options/publish.go#L95), and the `namer` is called for each tag [here](https://github.com/google/ko/blob/v0.8.0/pkg/publish/default.go#L140), so I think the flag documentation is just wrong. 🤔 

As an alternative, as mentioned in the original PR description, we could instead do `KO_DOCKER_REPO=quay.io/shipwright ko publish --base-import-paths ./cmd/manager` which would name the image after the Go package, `quay.io/shipwright/manager`.

Reply:
I think we should not rename the `shipwright/shipwright-operator`. I tested this change, it looks good to me, I can see this will give me a `docker pull quay.io/shipwright/shipwright-operator:latest` and `docker pull quay.io/shipwright/shipwright-operator:latest-debug`.

Reply:
Per @qu1queee's finding, I'm going to approve. We can address image namings later.